js/interactions.md

```diff
@@ -12,13 +12,50 @@ Ensure you have [installed and configured the Amplify CLI and library]({%if jeky
 
 AWS Amplify implements [Amazon Lex](https://aws.amazon.com/lex) as the default chatbots service. Amazon Lex supports creating conversational bots with the by the same deep learning technologies that power Amazon Alexa.
 
-## Create your Chatbot
+#### Automated Setup
+
+Run the following command in your project's root folder:
+
+```bash
+$ amplify add interactions
+```
+
+The CLI will lead you through the steps to specify the chatbot to be created. 
+
+You can choose to start from a sample chatbot or start from scratch.  If you choose to start from scratch, the CLI will prompt you with a series of questions to set the intents and slots for the chatbot. 
+
+You are allowed to run the `amplify add interactions` command multiple times to add multiple chatbots into your project.
+
+{The Interactions category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.}
```

Comment:
Something is buggy here - "{The Interactions category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.} {: .callout .callout--info}"